src/main/java/com/gmail/goosius/siegewar/playeractions/PlunderTown.java

```diff
@@ -79,7 +80,18 @@ public static void processPlunderTownRequest(Player player,
 			if(nationOfPlunderingResident != siege.getDefender())
 				throw new TownyException(Translation.of("msg_err_siege_war_cannot_plunder_without_victory"));
 
-			plunderTown(siege, townToBePlundered, (Nation)siege.getDefender());
+			switch (siege.getStatus()) {
+				case ATTACKER_WIN:
+					// Revolting Town won.
+					plunderGovernment(siege, (Nation)siege.getDefender(), townToBePlundered);
+					break;
+				case DEFENDER_WIN:
+					// Revolting Town lost.
+					plunderGovernment(siege, townToBePlundered, (Nation)siege.getDefender());
+					break;
+				default:
+			}
+			
```

Comment:
:construction: 
- The above logic update should be reverted
- This is because "plunder" is defined as the plunder of a town
- Thus if the attacker (i.e. the rebel town) wins....... well they wouldn't wish to plunder their own town.....

*note:*
- In other siege types, money can be gained by the non-plundering side. This is done via the "warchest" mechanic. But in Revolt revolt sieges even this money does not come into play, because the warchest size is always 0.
- To put it another way, in revolt sieges, the reward for the rebels is not money, but freedom.